helm/botkube/values.yaml

```diff
@@ -50,6 +50,24 @@ kubeconfig:
   ##    config: {base64_encoded_kubeconfig}
   existingSecret: ""
 
+## Format: actions.{alias}
+actions:
+  ## kubectl based action
+  'show-created-resource':
+    ## Kubectl action configuration - requires that a kubectl executor with correct permission is available
+    kubectl:
+      enabled: true
+      run: "kubectl get {{.Event.ResourceType}} -n {{.Event.Namespace}} {{.Event.Name}}"
```

Comment:
Another suggestion: Instead of wiring the actions per source, @huseyinbabal asked why not add top level bindings config.

Taking his idea and re-working it slightly, we can add the attributes below to the `action` definition:
```YAML
  resources: ["deployments", "pods", "namespaces", "services", "ingresses"]
  events: ["create"]
  sources: [".*"]
```

Now there is no need to add `action` to specific sources. Giving us one place to view and configure how automations affect target resource/event/source combos.

Reply:
Yep, my idea was totally a remake of prometheus configs. So prometheus has `routes` and `receivers`. Based on the criteria matching in `route` section, the command can be executed as defined in the `receivers` part. Example 👇 
```
  config:
    global:
      resolve_timeout: 5m
    route:
      routes:
      - match:
          event_type: create
        receiver: show-created-resource
    receivers:
    - name: 'show-created-resource'
      kubectl:
        enabled: true
        run: "kubectl get {{.Event.ResourceType}} -n {{.Event.Namespace}} {{.Event.Name}}"
``

Reply:
I reworked the `values.yaml` based on the discussion here.